backend/src/dependencies/db.py

```
@@ -62,21 +65,25 @@ def get_engine():
+                .where(User.role == UserRole.ADMIN)
+            ).first():
+                return
+            admin_user = User(
+                username="admin",
+                role=UserRole.ADMIN,
+                password_hash=hash_password("password"),
+                customer_id=None,
+            )
+            session.add(admin_user)
+            session.commit()
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)